client/components/RelatedProducts/RelatedProducts.jsx

```diff
@@ -4,6 +4,9 @@ import CustomOutfitList from './Outfit/CustomOutfitList.jsx';
 import Grid from '@material-ui/core/Grid';
 import axios from 'axios';
 import config from '../../../config.js'
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
```

Comment:
Consider using destructuring on the imports that all come from "core," (Grid, Snackbar, IconButton).